packages/smplayer.rb

```diff
@@ -0,0 +1,64 @@
+require 'package'
+
+class Smplayer < Package
+  description 'SMPlayer is a free media player for Windows and Linux with built-in codecs that can play virtually all video and audio formats.'
+  homepage 'https://www.smplayer.info/'
+  version '24.5.0'
+  license 'GPL-2'
+  compatibility 'x86_64'
+  source_url "https://github.com/smplayer-dev/smplayer/releases/download/v#{version}/SMPlayer-#{version}-x86_64.AppImage"
+  source_sha256 '85fb5a2322f48a298b7784fc3516e672525593c017ca504310bff05e1330457b'
+
+  no_compile_needed
+
+  depends_on 'gtk3'
+  depends_on 'gdk_base'
```

Comment:
Dependencies in alphabetical order?

Reply:
Let's not get too anal now. :)

Reply:
:woman_shrugging: It'll be fixed whenever I write the cop for it, so its not that big of a deal.

Reply:
I get that it makes sense to sort in alphabetical order to identify duplicate lines and that definitely makes sense for packages with a lot of dependencies.